src/core/qgsmapsettings.cpp

```diff
@@ -500,7 +500,7 @@ QgsPoint QgsMapSettings::layerToMapCoordinates( const QgsMapLayer *layer, QgsPoi
 {
```

Comment:
There is surely a good reason, but why in these methods the `point` argument is not a const reference?
I'd seen that in a recent past, and I wondered about.

Reply:
It should be fixed. In the past QgsPoint was a small and trivially copyable value, but it's not anymore.

Reply:
Done.